packages/client/src/components/form/ReviewActionComponent.tsx

```diff
@@ -377,7 +379,7 @@ class ReviewActionComponent extends React.Component<
                 size="large"
                 id="submit_form"
                 onClick={this.toggleSubmitModalOpen}
-                disabled={totalFileSizeExceeded}
+                disabled={!hasErrorsOnFields || totalFileSizeExceeded}
```

Comment:
This variable is a little confusing. The submit button is disabled if there are no errors on the fields? Shouldn't it be the other way around?